### DISCUSSION TITLE
实例的账号管理，当中的编辑功能的用意是什么呢？

### DISCUSSION THREAD

**Discussion by peng19832** [hhyo/Archery#2444] [Q&A]
如题：
将数据库账号的密码，录入平台数据库，好像没啥意义呢，也没有备忘查询的作用，这个按钮留在这里有什么考虑吗？
![image](https://github.com/hhyo/Archery/assets/9590209/2b547803-487a-43ff-9318-ebd32b13d29a)

**Comment by nick2wang** [comment]
应该就是作备忘用的，点账号进django后台就可以看到密码明文了
